### DISCUSSION TITLE
Building my blog

### DISCUSSION THREAD

**Discussion by Alex-Serafim** [community/community#128548] [New to GitHub]
### Select Topic Area

Question

### Body

I wanted to use my github on two computers, one at home and one at work.
I have a blog, and I don't know how to work on it on two computers, I only know how to work with one.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Alex-Serafim , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by yilmazemrecom** [comment]


**Reply by queenofcorgis** [reply]
@emreylmzcom
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by nminhducit** [comment]
Not very specific, bro
